test/extensions/transport_sockets/tls/context_impl_test.cc

```diff
@@ -62,6 +62,16 @@ TEST_F(SslContextImplTest, TestVerifySubjectAltNameDNSMatched) {
   EXPECT_TRUE(ContextImpl::verifySubjectAltName(cert.get(), verify_subject_alt_name_list));
 }
 
+TEST_F(SslContextImplTest, TestMatchSubjectAltNameDNSMatched) {
```

Comment:
Where are you testing the old behavior now? I don't see any new deprecated feature tests?

Reply:
New tests I have added in this class to test the new behaviour. For config test, this test https://github.com/envoyproxy/envoy/blob/master/test/server/listener_manager_impl_test.cc#L174 which is already deprecated, tests the old behaviour. Am I missing some thing else?

Reply:
I have added a new test for `TlsTransportSocket` as well that runs with old config, if that is what you were looking for. PTAL